src/content/docs/cachyos_basic/why_cachyos.md

```diff
@@ -3,11 +3,13 @@ title: Why CachyOS?
 description: Why CachyOS may be better for you
 ---
 
-CachyOS is an user-friendly and highly optimized Linux distribution based on Arch Linux.
+CachyOS is a user-friendly and highly optimized Linux distribution based on Arch Linux.
 
-# What makes CachyOS not just Arch based distro.
+## What makes CachyOS not just another Arch based distro?
```

Comment:
This line seems a bit out of place, maybe we can merge it with the bottom paragraph